### PR TITLE
Prosopo api error error code fixes

### PR DESCRIPTION
**Pull request by HughParry**

### CODE REVIEW

File: packages/provider/src/api/authMiddleware.ts [hunk 1]
```diff
@@ -46,12 +46,14 @@ const extractHeaders = (req: Request) => {
 
     if (!signature || !blocknumber) {
         throw new ProsopoApiError('CONTRACT.INVALID_DATA_FORMAT', {
-            context: { error: 'Missing signature or block number' },
+            context: { error: 'Missing signature or block number', code: 400 },
```
Comment (goastler) [inline packages/provider/src/api/authMiddleware.ts:49]: you've used status on other lines, is it status or code?

**Review by goastler — CHANGES_REQUESTED** [review]

File: packages/provider/src/api/authMiddleware.ts (hunk 1 above)
Comment (HughParry) [reply packages/provider/src/api/authMiddleware.ts:49]: Should be code everywhere. Status incorrectly used in Flux error handler. Good catch

**Review by goastler — APPROVED** [review]

**Comment by goastler** [comment]
@prosoponator u alive?

**Comment by goastler** [comment]
@prosoponator approve

**Review by prosoponator — APPROVED** [review]

**Comment by goastler** [comment]
@prosoponator disapprove

**Pull request merged**